### PR TITLE
Fixes violations of and reenables rubocop Style/EmptyLinesAroundModuleBody.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Fixes generated by `bundle exec rubocop --only Style/EmptyLinesAroundModuleBody --auto-correct`.

### CODE REVIEW

File: pegasus/src/env.rb [hunk 1]
```diff
@@ -27,8 +27,6 @@ def src_dir(*paths)
   pegasus_dir('src', *paths)
 end
 
-module Pegasus
-
```
Comment (islemaster) [inline pegasus/src/env.rb:31]: Is this a mistake?

Comment (ashercodeorg) [inline pegasus/src/env.rb:31]: Yep (from a bad merge on my part). Good catch.

**Comment by islemaster** [comment]
LGTM

**Comment by ashercodeorg** [comment]
Note that the failing tests are a known (fixed) issue. Doesn't seem worth it to merge the fix and rerun tests, so I'm not going to do so.

**Pull request merged**